contracts/SSVNetwork.sol

```diff
@@ -216,6 +216,10 @@ contract SSVNetwork is
         _delegate(SSVStorage.load().ssvContracts[SSVModules.SSV_CLUSTERS]);
     }
 
+    function exitValidator(bytes calldata publicKey, uint64[] calldata operatorIds) external override {
```

Comment:
@lior-blox i believe we missing a param here of withdraw address thats wanted.  I dont think its good that we assume owner wants to receive to this address, especially with DKG, i assume with DKG that some validators will be created with some sort of multisig and maybe they dont want to withdraw there.

Reply:
This is irrelevant as we are not setting the withdraw address anyway.